readme.md

```diff
@@ -19,6 +19,7 @@ Contributions welcome. Add links through pull requests or create an issue to sta
   - [Articles](#articles)
   - [Books](#books)
   - [Communities](#communities)
+  - [Newsletters](#newsletters)
```

Comment:
Good suggestion for a new section.

Please keep headings in alphabetical order.